### ISSUE TITLE
Find/Fix unknown issues with Heroku deploy

### ISSUE THREAD

**Issue by dwieringa** [citizenlabsgr/adopt-a-drain#4]
Jace launched the code we have in github (SF's code) on Heroku.  Currently the map doesn't show.  We need to analyze the app to determine what else is needed to get it running "fully" (at least enough to demo).

**Comment by dwieringa** [comment]
I'm planning to jump on this, but don't expect to have time within the next week.  I'll update this when I do.

**Comment by jacebrowning** [comment]
@dwieringa Did everything load correctly when run locally?

**Comment by Allen616** [comment]
I am going to review the data and format needed and will request from LGROW.

**Comment by dwieringa** [comment]
The SF app is now running on as Review App on the Heroku account.  GitHub PR #13.  https://lgrow-staging-pr-13.herokuapp.com/

The issues we ran in to before was that the assets need to be precompiled (listed in deploy instructions).  I also had to create a Google Maps API Key and set it in Heroku.  When things weren't working, I also set `config.serve_static_assets = true` (per Heroku Rails docs) and removed the `asset_host` setting which was forcing to SF's server.

In hindsight, I should probably review that everyone of these steps was required.
